CMakeLists.txt

```diff
@@ -491,6 +528,11 @@ if(ROCALUTION_FOUND)
   target_include_directories(opmsimulators PUBLIC ${rocalution_INCLUDE_DIR}/rocalution)
 endif()
 
+if(rocsparse_FOUND AND rocblas_FOUND)
+  target_link_libraries( opmsimulators PUBLIC rocsparse )
+  target_link_libraries( opmsimulators PUBLIC rocblas )
```

Comment:
Please use the targets roc::rocsparse and roc::rocblas instead here.

Reply:
This might solve the linker problem on Jenkins.

Reply:
67ad9e6